packages/costa/test/runtime_test.ts

```diff
@@ -69,48 +102,24 @@ describe('create a costa runtime', () => {
     expect(collectCsvOnBare.version).toBe(collectCsvLatest.version);
   });
 
-  it('should install the package without conda packages', async () => {
-    await costa.install(collectCsv, process);
-    await stat(path.join(
-      costa.options.installDir,
-      'node_modules',
-      collectCsv.name
-    ));
-  });
-
-  it('should install the package with conda packages', async () => {
```

Comment:
Ditto, we need a reason to remove this case.

Reply:
https://github.com/alibaba/pipcook/pull/518/files#diff-bdea6cb8b04e910bac8851b60725bbaeL30